tests/integration/targets/azure_rm_roleassignment/aliases

```diff
@@ -1,3 +1,2 @@
 cloud/azure
 destructive
-unsupported
```

Comment:
@paultaiton If you remove the "unsupported" key, Please add "shippable/azure/group10" for this module! Thank you very much!

Reply:
Can do, thanks @Fred-sun